python/lsst/cell_coadds/_identifiers.py

```diff
@@ -163,11 +158,10 @@ def from_data_id(cls, data_id: DataCoordinate) -> ObservationIdentifiers:
         identifiers : `ObservationIdentifiers`
             Struct of identifiers for this observation.
         """
-        # The 'type: ignore' directives below are present because we require a
-        # a fully-expanded data ID, but this isn't embedded in the types.
+        packer = Instrument.make_default_dimension_packer(data_id, is_exposure=False)
```

Comment:
Ah! This is the method I was looking for without having to configure the dataIds arbitrarily.